### DISCUSSION TITLE
Reformatting annotations to train recognition model

### DISCUSSION THREAD

**Discussion by zohaib-khan5040** [mindee/doctr#1278] [Q&A]
Hello!

I have a dataset with annotations similar to the YOLO format - for each image I have a list of bounding boxes, each having coordinates for x,y, and the width and height. I wish to make annotations similar to what is provided in the `references/` folder for finetuning detection and recognition models.

I can get the detection models to work, that's not an issue. I do want to ask how I can manipulate these annotations to get a structure for the recognition dataset. I was initially thinking of getting a reading order for the boxes but I doubt that's a good way to go about it.

Thank you! :pray:

**Comment by zohaib-khan5040** [comment]
Found this function in the repo:
https://github.com/mindee/doctr/blob/95857cf05d7b8054fc1dcb2a20d79119d50488ba/doctr/models/builder.py#L43

Still curious if this is the best approach for something like this.

**Comment by felixdittrich92** [comment]
Hi @zohaib-khan5040 👋, 

Not sure what you want to reach 😅 but the order doesn't matter if you want to train a recognition model (it's enough to provide the Image crops and coresponding labels)
So could you explain your problem a bit more please ? :)
